README.md

```diff
@@ -9,6 +9,18 @@ Measurement Operator Trainer is a Python package training measurement operators
 
 Please refer to the [MOtrainer documentation](https://vegewaterdynamics.github.io/motrainer/) for more details.
 
+## Installation
+
+MOTrainer requires Python version `>=3.10`. It can be installed from PyPI: 

```

Comment:
I suggest you follow the format you have in the README, i.e.:

Note: MOTrainer requires Python version `>=3.10`.

MOTrainer can be installed from PyPI: [...]

While unlikely, the sentence as is is ambiguous in what `It` refers to.